### ISSUE TITLE
Add documentation for bingo_config 

### ISSUE THREAD

**Issue by AATDev21** [epam/Indigo#655]
There isn't any documentation for Bingo setup and related options.
Would be convenient to have something like this page for Indigo Toolkit:

https://lifescience.opensource.epam.com/indigo/options/index.html

**Comment by dan2097** [comment]
Do you mean like https://lifescience.opensource.epam.com/bingo/bingo-oracle.html ?
(there are also corresponding pages for the PostgreSQL, SQL Server and NoSQL implementations, but the detailed options are explained in the Oracle implementation's help pages)

**Comment by AATDev21** [comment]
@dan2097, no, I mean general Bingo configuration and all options used for it (like on the picture below). I haven't found anything about it in docs.

<img width="342" alt="image" src="https://user-images.githubusercontent.com/78915761/154481324-5a999a42-4c68-49d5-91d4-4d0138649615.png">
